### PR TITLE
fix: move to conditional wrapper

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- We used a dynamic render, but should have use our conditional render instead since it would simply render a empty fragment.

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1837 #done

### CODE REVIEW

File: packages/shared/src/components/widgets/PostUsersHighlights.tsx [hunk 1]
```diff
@@ -78,15 +79,22 @@ const Image = (props: SourceAuthorProps) => {
 const UserHighlight = (props: SourceAuthorProps) => {
   const { id, handle, name, username, permalink, userType = 'source' } = props;
   const Icon = getUserIcon(userType);
-  const LinkWrapper = userType === 'source' ? React.Fragment : ProfileTooltip;
+  const userTypeNotSource = userType === 'source';
```
Comment (sshanzel) [inline packages/shared/src/components/widgets/PostUsersHighlights.tsx:82]: I think there was a typo.
```suggestion
  const userTypeNotSource = userType !== 'source';
```

**Review by sshanzel — CHANGES_REQUESTED** [review]
Looks great! Definitely one to use the ConditionalWrapper. Just needs to have the right condition.

File: packages/shared/src/components/widgets/PostUsersHighlights.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/widgets/PostUsersHighlights.tsx]: Ah yes my bad was testing it and accidently pushed the wrong one 🙈

**Review by sshanzel — APPROVED** [review]
Let's go 🚀

**Review by capJavert — APPROVED** [review]

**Pull request merged**